inbm/dispatcher-agent/dispatcher/sota/sota.py

```diff
@@ -372,6 +372,9 @@ def execute_from_manifest(self,
                 self._dispatcher_broker.send_result(SOTA_FAILURE)
                 raise SotaError(SOTA_FAILURE)
 
+    def _is_reboot_device(self) -> bool:
```

Comment:
<h3>$\color{#FFA500}Improvements: Code Maintainability$</h3>

The method `_is_reboot_device` could be refactored into a property for better readability and to follow Pythonic conventions.